website/src/e-commerce/src/widgets/index.js

```diff
@@ -0,0 +1,13 @@
+export { default as brands } from './Brands';
```

Comment:
I'm not sure if this pattern is really necessary in an example like this

Reply:
This is fixed in 49167efc5e365472fd851b5971e7a233e6319c63.

Reply:
What would you have used? It's to export them all from `widgets/index.js`.

Reply:
since all widgets are always mounted, I would've gone for the regular way of calling the functions inline in the main file. I understand why you split it up here, but I fear like it might confuse people

Reply:
The goal of the demo is also to educate people how to build apps with InstantSearch. I tried several patterns and this once seem like an easy structure to work with. It will be easy to migrate this demo to React/Vue/etc. 

We've often neglected how to integrate InstantSearch into apps and most of our examples don't make usage of modern JavaScript feature that we usually like working with.